src/bitbucket.js

```diff
@@ -175,13 +175,18 @@ const listFilesOfInterestInRepoUsingCache = mem(
 const isNotASymbolicLink = item => !item.attributes.some(attribute => attribute === 'link')
 
 const getContentsOfFileByDownloadUrl = async (username, appPassword, downloadUrl) => {
+  console.debug(`Getting contents of ${downloadUrl}`)
   const response = await request(`GET ${downloadUrl}`, {
     headers: {
       authorization: "Basic " + Buffer.from(`${username}:${appPassword}`).toString('base64'),
     },
   })
   return response.data
 }
+const getContentsOfFileByDownloadUrlUsingCache = mem(
+    getContentsOfFileByDownloadUrl,
+    { cacheKey: arguments => arguments[2] }
```

Comment:
The `arguments[2]` refers to the third argument to `getContentsOfFileByDownloadUrl(username, appPassword, downloadUrl)`, so that subsequent calls for that same downloadUrl use the cache.